Gemfile.template

```diff
@@ -27,3 +27,4 @@ gem "rack-test", :require => "rack/test", :group => :development
 gem "rspec", "~> 3.5", :group => :development
 gem "webmock", "~> 3", :group => :development
 gem "murmurhash3", "= 0.1.6" # Pins until version 0.1.7-java is released
+gem "logstash-integration-kafka", "~>10.12.1"
```

Comment:
If we introduce this here we should also add:

```diff
diff --git a/Gemfile.jruby-2.5.lock.release b/Gemfile.jruby-2.5.lock.release
index 55e9fcc03..ca110f1e3 100644
--- a/Gemfile.jruby-2.5.lock.release
+++ b/Gemfile.jruby-2.5.lock.release
@@ -877,7 +877,7 @@ DEPENDENCIES
   logstash-input-unix
   logstash-integration-elastic_enterprise_search
   logstash-integration-jdbc
-  logstash-integration-kafka
+  logstash-integration-kafka (~> 10.12.1)
   logstash-integration-rabbitmq
   logstash-output-cloudwatch
   logstash-output-csv
```

Reply:
done